packages/app/src/components/DBRowTable.tsx

```diff
@@ -983,23 +992,117 @@ export function DBSqlRowTable({
     }
   }, [isError, onError, error]);
 
+  const patternColumn = columns[columns.length - 1];
+  const groupedPatterns = useGroupedPatterns({
+    config,
+    samples: 10_000,
+    bodyValueExpression: patternColumn ?? '',
+    totalCount: undefined,
+    enabled: denoiseResults,
+  });
```

Comment:
Just a comment for later: If we notice performance issues here, we could move this to a webworker